Makefile

```
@@ -160,6 +161,9 @@ get-manifests: ## Fetch components manifests from remote git repo
 	./get_all_manifests.sh
 CLEANFILES += odh-manifests/*
 
+.PHONY: create-api-docs
+create-api-docs: crd-ref-docs ## Creates API docs using https://github.com/elastic/crd-ref-docs
```

nit, but I think `make api-docs` sounds a bit better :)

```suggestion
.PHONY: api-docs
api-docs: crd-ref-docs ## Creates API docs using https://github.com/elastic/crd-ref-docs
```